components/director/internal/domain/application/service.go

```diff
@@ -514,3 +533,55 @@ func (s *service) ensureIntSysExists(ctx context.Context, id *string) (bool, err
 	log.C(ctx).Infof("Integration System with id %s exists", *id)
 	return true, nil
 }
+
+func (s *service) getScenariosAndRuntimes(ctx context.Context, tenant, applicationID string) ([]string, []string, error) {
+	log.C(ctx).Infof("Getting formations for application with id %s", applicationID)
```

Comment:
formation is not a word that is used in the code (i just saw a couple of tests that use it) - every where else its called scenario. So for consistency use scenarios